ariac-server/ariac-server/Dockerfile

```diff
@@ -1,86 +1,37 @@
-FROM ubuntu:bionic
+ARG BASE_DIMG
+FROM $BASE_DIMG
```

Comment:
Wouldn't this make a Docker image tag always need to be passed, which may seem inconvenient?